Before: ee8716bdbcf4169340b744e4ffcd5d9c94a885b343d878b5e1cbc5f4e60e80d3
After: 024467d03e7db9ef40534ffaec1a1fe2233dfb1248e12692445be914b5f514af
Patch: 56092596c5547f42ad4b283cecde46a91a4e7b8acaf0065a292f1dff2c8035d6
--- corvid-features/corvid.rb	2012-08-09 12:37:41.761933580 +1000
+++ corvid-features/corvid.rb	2012-08-09 12:37:54.512006181 +1000
@@ -3,11 +3,4 @@
 install {
   copy_file       'Rakefile'
   empty_directory 'lib'
-  empty_directory 'wtffff'
-}
-
-update {|ver|
-  if ver == 2
-    #empty_directory 'wtffff'
-  end
 }
